convert2rhel/unit_tests/main_test.py

```diff
@@ -174,6 +203,8 @@ def test_pre_ponr_conversion_order_with_rhsm(self):
     @mock_calls(subscription, "disable_repos", CallOrderMocked)
     @mock_calls(subscription, "enable_repos", CallOrderMocked)
     @mock_calls(subscription, "rename_repo_files", CallOrderMocked)
+    @mock_calls(utils, "parse_version_number", ParserVersionMocked)
```

Comment:
Use `CallOrderMocked` to be able to test for call order, then expand the `OrderedDict` in the test
```suggestion
    @mock_calls(subscription, "download_subscription_manager_packages", CallOrderMocked)
```